src/tooltip.js

```diff
@@ -44,6 +44,9 @@
             ,   enabled = true  //True -> tooltips are rendered. False -> don't render tooltips.
             ,   duration = 100 // duration for tooltip movement
             ,   headerEnabled = true
+            ,   colorDivEnabled = true //enable color div in tooltip for legend elements
```

Comment:
Not very fond of this name, what if we wanted to change it to a span or anchor tag in the future?   Also, it's not really enabling functionality, it's just showing them... so maybe name them like "showColors", "showKey", and "showValues" ?